assets/lokomotive-kubernetes/bootkube/resources/charts/lokomotive/templates/mutating-webhook.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: webhook-admission
+  labels:
+    name: webhook-admission
+webhooks:
+  - name: mutating.kinvolk.io
+    clientConfig:
+      caBundle: "{{ .Values.certificates.caCert }}"
+      service:
+        name: webhook-admission
+        namespace: kube-system
+        path: /mutate
+    rules:
+      - operations: ["CREATE"]
+        apiGroups: [""]
+        apiVersions: ["v1"]
+        resources: ["serviceaccounts"]
+    sideEffects: None
+    failurePolicy: Ignore
+    admissionReviewVersions: ["v1", "v1beta1"]
```

Comment:
Shouldn't this be just `v1beta1` ?

Reply:
Webhook code has `v1` response, but added v1beta1 for any backward compatibility, in fact this can be removed

Reply:
Ah I see.

Reply:
I think we don't need backward compatibility, as this is not intended to run outside of L8e clusters.